inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
```

Comment:
Update documentation here: if you agree with my suggestion of returning the ```s_dev2``` directly, the docstring can stay the same. If you think a dictionary object is more appropriate, change the documentation to say the exact type of output returned from the data!